vitrina/datasets/models.py

```diff
@@ -94,13 +96,16 @@ class Dataset(TranslatableModel):
     access_rights = models.TextField(blank=True, null=True, verbose_name=_('Prieigos teisės'))
     distribution_conditions = models.TextField(blank=True, null=True, verbose_name=_('Platinimo salygos'))
 
-    tags = tagulous.models.TagField(
+    tags = TagField(
         blank=True,
         force_lowercase=True,
         space_delimiter=False,
+        autocomplete_view='autocomplete_tags',
         autocomplete_limit=20,
         verbose_name="Žymės",
         help_text=_("Pateikite kableliu atskirtą sąrašą žymių."),
+        autocomplete_settings={"width": "100%"},
```

Comment:
Is this really fixed? I still see tag items out of tag field:

![image](https://user-images.githubusercontent.com/297583/207908079-2e2777cf-d2c3-4991-9dae-f3b3fc682c14.png)

Reply:
By the way, this only appears using Firefox, it works correctly with Chrome.

Reply:
Maybe you didn't rebuild webpack static files? For me it looks like this:

![image](https://user-images.githubusercontent.com/107176624/208371412-2c57e469-f339-49fc-a3ba-c650df0afd64.png)